### ISSUE TITLE
Change default version to 3.0

### ISSUE THREAD

**Issue by hatifnatt** [saltstack-formulas/zabbix-formula#87]
Formula default version is 2.2 which is currently under Limited Support since August, 2017 till August, 2019.
It's time to change default version to 3.0.
Info about dates from [Zabbix Life Cycle & Release Policy](https://www.zabbix.com/life_cycle_and_release_policy).

**Comment by pauldalewilliams** [comment]
I agree.  We really need to have support for the latest supported versions, which you and I are working on.  (By the way, I am planning to get back around to your PRs.  Just away at a tech conference this week with limited time.  Should be able to get to it next week.)
